### ISSUE TITLE
Review notes

### ISSUE THREAD

**Issue by chrisvfritz** [snurby7/RailsAppV2#1]
This is looking mostly awesome! There's a very evident attention to detail. :tada: :smiley: There are a couple of items I'd like you to update though, to even further improve:

This first one is purely stylistic. These four lines here:

``` ruby
validates_presence_of :title
validates_presence_of :author
validates_presence_of :review
validates_presence_of :studio
```

can be shortened to:

``` ruby
validates_presence_of :title, :author, :review, :studio
```

The second is more functional. When I enter an invalid review score, I get a message saying simply `Invalid Review Score`. Instead, could you update it to let the user know what *is* a valid review score as well?

### Optional improvements

Since last time we worked on tightening Bootstrap integration, [here's a gem](https://github.com/bootstrap-ruby/rails-bootstrap-forms) that can help you tighten Bootstrap integration in your forms without too much extra work, while at the same time slimming down your form markup! :smiley:

**Comment by snurby7** [comment]
Fixed the issues in commit 6611e1f491 as well as the optional improvement. It had to be done, my OCD was going to kick in. Still not perfect, but much better looking forms.

**Comment by chrisvfritz** [comment]
:+1: Very nice! :tada:
